qark/plugins/manifest/allow_backup.py

```diff
@@ -16,16 +17,20 @@ def __init__(self):
                                          "http://developer.android.com/reference/android/R.attr.html#allowBackup"))
         self.severity = Severity.WARNING
 
-    def run(self, file_object):
+    def run(self, files, extras=None):
```

Comment:
I think we should call this context - conveys more meaning methinks

Reply:
Personally `context` doesn't mean anything to me, but `extras` gives more hints about what the variable is containing. Perhaps something more verbose like `useful_variables` is even better. @neilatli what say you?

Reply:
I'm primed to consider `context` to be correct here. But throwing out all previous experience, perhaps a name like `apk_constants` better conveys what this variable is meant to hold.

Reply:
@dhaugh how do you feel about `apk_constants` or `apk_info`?

Reply:
apk_constants works